### PR TITLE
Added Tumult Hype 3.5 JS

### PR DESCRIPTION
**Pull request by dimitrigq**

### CODE REVIEW

**Comment by megawac** [comment]
I'm sorry, but this request looks like its requesting a library that is specific to a single application and not an open source library. The purpose of jsDelivr is to host open source libraries which can be reused across multiple projects. As such, we generally require new projects to have at minimal: some documentation, a source repository or website to download the code and some general reusability.

Feel free to leave a comment on this pull request if you disagree.

**Comment by dimitrigq** [comment]
Tumult Hype3 is an Html5 animation and timeframe application which is widely used for interactive web applications. We've just launch the first html5 banners from the app. The code can be used amongst multiple projects.

**Comment by megawac** [comment]
Is there a docs page we can reference? http://wefuse.com/ seems to be an empty page

**Comment by dimitrigq** [comment]
wefuse.com is just the agency working on the banners. We just use the app Tumult Hype 3 for banner creation. The company that makes the application is tumult.com/hype3. I can update the info file to ref# their documentation.

File: files/hype3/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "We Fuse"
+github = "https://github.com/WeFuse/"
```
Comment (megawac) [inline files/hype3/info.ini]: this should be `wefuse/hype`

**Comment by megawac** [comment]
please do

**Pull request closed without merging**